tests/31sync/15lazy-members.pl

```diff
@@ -99,9 +99,21 @@
             )
          }, foreach => [ 1 .. 10 ])
       })->then( sub {
+         # Ensure synapse has propagated this message to Alice's synchotron
+         await_sync_timeline_contains( $alice, $room_id, check => sub {
+            my ($event) = @_;
+            log_if_fail "event", $event;
+            my $result = $event->{type} eq "m.room.message" &&
+               $event->{sender} eq $charlie->user_id &&
+               $event->{content}->{body} eq "Message 10";
+            $result
```

Comment:
Is this the equivalent of `return $result;`, similar to rust?

Reply:
Aha, OK. Thanks for the explanation. Also looks like there is no differentiator through the semicolon as I originally thought.